locations/spiders/gridserve_gb.py

```
@@ -1,33 +1,25 @@
+from scrapy.http import Response
+from scrapy.spiders import SitemapSpider
 
 from locations.categories import Categories, apply_category
 from locations.items import Feature
-from locations.json_blob_spider import JSONBlobSpider
 
 
-class GridserveGBSpider(JSONBlobSpider):
+class GridserveGBSpider(SitemapSpider):
     name = "gridserve_gb"
     item_attributes = {"operator": "Gridserve", "operator_wikidata": "Q89575318"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)